parsl/tests/test_python_apps/test_pluggable_future_resolution.py

```diff
@@ -29,14 +30,41 @@ def test_simple_pos_arg():
 
 
 @parsl.python_app
-def b_first(x: Sequence[int]):
+def b_first(x: Iterable[int]):
     return x[0] + 1
```

Comment:
you can't do [0] on an `Iterable` - mypy detects that and is giving an error here...
an Iterable style implementation here would be to call `next(iter(x))` I think.

Reply:
Oops, good catch. I didn't mean to change this one.